src/azul/service/elasticsearch_service.py

```diff
@@ -525,21 +525,65 @@ def transform_summary(self,
                 'sum',
                 field='contents.cell_suspensions.total_estimated_cells_'
             )
-            # Add a total cell count aggregate
+            # FIXME: Remove deprecated field totalCellCount
+            #        https://github.com/DataBiosphere/azul/issues/3650
+            # Add a cell suspensions cell count aggregate
             es_search.aggs.metric('totalCellCount',
                                   'sum',
                                   field='contents.cell_suspensions.total_estimated_cells_')
+            # Add a cell suspensions cell count aggregate from projects
```

Comment:
The two hunks (this one and the one below) are very similar now. Try to extract a function. As part of this consolidation, we should also consider relying on the field type again.